pkg/cmd/roachtest/cluster.go

```diff
@@ -52,18 +53,21 @@ const (
 )
 
 var (
-	local        bool
-	cockroach    string
-	cloud                     = gce
-	encrypt      encryptValue = "false"
-	instanceType string
-	workload     string
-	roachprod    string
-	buildTag     string
-	clusterName  string
-	clusterWipe  bool
-	zonesF       string
-	teamCity     bool
+	local            bool
```

Comment:
there is a whole bunch of work that has gone on in this file - https://github.com/cockroachdb/cockroach/blame/master/pkg/cmd/roachtest/cluster.go

not sure how to provide any isolated context to the changes.